public.updatesite/build/update_version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This is a duplicate of the script in /build, why?

Reply:
Oops, that was a change I forgot to ignore in github when I was working in eclipse. That can be removed.